src/index.ts

```diff
@@ -17,16 +17,9 @@ import { TracingMode, enableTracing } from "./tracing";
 import { redirectHandlers } from "./wrapper";
 import { addCloudWatchForwarderSubscriptions, addExecutionLogGroupsAndSubscriptions } from "./forwarder";
 import { addOutputLinks, printOutputs } from "./output";
-import { FunctionDefinition } from "serverless";
 import { setMonitors } from "./monitors";
 import { getCloudFormationStackId } from "./monitor-api-requests";
 
-// Separate interface since DefinitelyTyped currently doesn't include tags or env
```

Comment:
Curious about why we can delete the interface here. Is it unnecessary?

Reply:
The version we currently have of DefinintelyTyped now supports it. I guess the interface was created when it wasn't.